tools/buildsys/src/builder.rs

```diff
@@ -211,17 +223,60 @@ impl VariantBuildArgs {
     }
 }
 
+struct VariantRepackArgs {
```

Comment:
nit: consistent naming (albeit long)

```suggestion
struct RepackVariantBuildArgs {
```